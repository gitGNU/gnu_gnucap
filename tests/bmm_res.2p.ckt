# semiconductor "resistor" test
.param pw=2u pl=1u pt=37
v1 1 0 dc 2 ac 5
r1 1 2 1k
r2 2 0 t1 (w=pw l=pl temp=pt)
.model t1 r rsh=1k tc1=.1
.print op v(nodes) r(r*)
.print ac v(nodes) 
.op
.ac
.list
.end
