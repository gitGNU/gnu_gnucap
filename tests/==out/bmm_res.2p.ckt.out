# semiconductor "resistor" test 
#           v(1)       v(2)       r(r1)      r(r2)     
 27.        2.         1.         1.K        1.K       
#Freq       v(1)       v(2)      
 0.         5.         2.5       
v1 ( 1 0 )  DC  2.  AC  5.
r1 ( 1 2 )  1.K
r2 ( 2 0 ) t1 l=pl w=pw temp=pt
.model t1 r ( rsh=1k narrow=NA( 0.) defw=NA( 1.u) tc1=.1 tc2=NA( 0.) tnom=NA( 27.))
